tests/auth_api/oidc/test_users.py

```diff
@@ -127,9 +121,11 @@ def test__create_user__when_a_user_with_same_tin_already_exists__it_should_not_i
         token_subject: str,
         id_token_encrypted: str,
     ):
-        # Attempts to add a user with the same tin as an existing user
-        # It should not add another user
-        
+        """
+        Attempts to add a user with the same tin as an existing user.
+        It should not add another user
```

Comment:
```suggestion
        It should not add another user

```

Newline between summary and extended summary.